CHANGELOG.rst

```diff
@@ -1,28 +1,9 @@
- | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
- | # # #
- | # # # Author:
- | # # # Naval Research Laboratory, Marine Meteorology Division
- | # # #
- | # # # This program is free software: you can redistribute it and/or modify it under
- | # # # the terms of the NRLMMD License included with this program. This program is
- | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
- | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
- | # # # for more details. If you did not receive the license, for more information see:
- | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+.. dropdown:: Distribution Statement
+
+ | # # # This source code is protected under the license referenced at
+ | # # # https://github.com/NRLMMD-GEOIPS.
 
 Please see
-``$GEOIPS_PACKAGES_DIR/geoips/CHANGELOG_TEMPLATE.rst``
-for instructions on updating CHANGELOG appropriately
+https://github.com/NRLMMD-GEOIPS/geoips/blob/main/CHANGELOG_TEMPLATE.rst
+for instructions on updating release notes appropriately
```

Comment:
Do not merge until pointing to main.

Also taking out everything from CHANGELOG.rst except pointing to CHANGELOG_TEMPLATE.rst, we can just have that never change.